### DISCUSSION TITLE
Font Size capabilities

### DISCUSSION THREAD

**Discussion by cloverLynn** [charmbracelet/lipgloss#378] [Ideas]
**Is your feature request related to a problem? Please describe.**
Not as much of a problem as a nice to have. For like Menu Headers

**Describe the solution you'd like**
Not Sure if there is a way to do this but some way to have a different size font would be really useful

**Describe alternatives you've considered**
Considered using something like [Figlit](https://github.com/patorjk/figlet.js/blob/main/README.md) to create ASCII titles on the fly but feels a bit resource heavy

**Additional context**
Love the tools y'all make <3

**Comment by bashbunni** [comment]
Hey @cloverLynn thanks for reaching out! Unfortunately it's not possible to change the font size when printing to the terminal. We've had this request before in [Glow](https://github.com/charmbracelet/glow/issues/291)

Sorry about that! You may opt to create ASCII titles or use special characters as they mentioned in the issue I linked.

I'm going to convert this to a discussion to keep the conversation open :)
